content/roadmaps/114-defi/content/111-yield-optimizers/107-convex.md

```diff
@@ -1 +1,6 @@
 # Convex
+
+Convex Finance is a platform for and Curve liquidity providers to earn additional interest rewards and Curve trading fees on their tokens. Put another way, Convex Finance offers boosted Curve staking.
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://docs.convexfinance.com/convexfinance/general-information/why-convex'>Convex for Curve.fi</BadgeLink>
```

Comment:
Some article with deeper explanation of Convex would do much better here 😉

Reply:
Updated.